alpha/books/RayTracingInOneWeekend.html

```diff
@@ -2855,7 +2872,7 @@
 
 Mirrored Light Reflection
 --------------------------
-For smooth metals the ray won’t be randomly scattered. The key math is: how does a ray get
+For a polished mirror the ray won’t be randomly scattered. The key question is: How does a ray get
```

Comment:
To me "mirror" implies a perfect reflector, making "polished" redundant (though I think we're discussing _imperfect_ reflectors here. How about "shiny materials" or "shiny metal"?

Reply:
I specifically went away from "metals" in the first sentence because (even though the chapter is called "metals") most everyday metals are not mirror-like

And that's ultimately what we're modeling here.

Let me think on it

Reply:
Going with polished metals. Tell me if you disagree with that

Reply:
Looks good.